terraform/environments/corporate-staff-rostering/locals_preproduction.tf

```diff
@@ -574,7 +574,7 @@ locals {
         access_logs     = false
 
         instance_target_groups = {
-          pp-csr-w-56-80 = {
+          pp-csr-w-34-80 = {
```

Comment:
This should still be 56 - the target groups follow the naming of the underlying webservers, whereas the LB should follow naming of the front end URLs.

Reply:
Oooooohhh... Right will do 👍